### ISSUE TITLE
Why does UCP PUT/GET not support iov ?

### ISSUE THREAD

**Issue by chenyidu** [openucx/ucx#6912]
It looks like the one-side operation API of UCT supports iov, so why UCP API  remove this part ? Is there any special consideration? Thanks in advance.
https://github.com/openucx/ucx/blob/d6d412e35d1541c8b9b05004c4113c3b1a639c03/src/uct/api/uct.h#L2779-L2782

**Comment by yosefe** [comment]
Right, currently UCP RMA operations do not support iov since it was not implemented. RMA supports only contiguous buffer since it the common use case for SHMEM/PGAS applicaitons.

**Comment by chenyidu** [comment]
> Right, currently UCP RMA operations do not support iov since it was not implemented. RMA supports only contiguous buffer since it the common use case for SHMEM/PGAS applicaitons.

@yosefe When will it be supported? Is there a plan?
